assets/css/style.scss

```diff
@@ -203,20 +205,24 @@ div.page-toc {
     margin: 0;
     width: 100%;
     position: -webkit-sticky;
-    position: sticky; 
+    position: sticky;
     top: 120px;
 }
 
 div.row.page-toc > div.toc-title {
     font-size: 1.1em;
-    font-weight: bold;    
+    font-weight: bold;
     padding: 20px 18px 20px;
     text-align: center;
 }
 
 div.row.page-toc > div.col-12.toc-content {
     margin: 0;
-    padding: 0;    
+    padding: 0;
+}
+
+body > div.row > div.section.col-sm-12 {
+    max-width: 75%; // must match the max-width of .header
```

Comment:
Comment says it must match the max-width of .header, but that one is 25%. Or does the comment mean it must be complementary?

Reply:
You're right.  I meant "complement".  I fixed the comment.